Sources/PackageRegistryTool/PackageRegistryTool+Sign.swift

```diff
@@ -18,6 +18,7 @@ import Basics
 import CoreCommands
 import PackageSigning
 import TSCBasic
+@_implementationOnly import X509 // FIXME: need this import or else SwiftSigningIdentity init at L128 fails
```

Comment:
👀

Reply:
I thought it was caused by the code leaking X509 error types but addressing that doesn't fix this